docker-test.hcl

```diff
@@ -0,0 +1,83 @@
+param "test-case" {}
+
+param "converge-bin-dir" {}
+
+param "image-name" {
+  default = "ubuntu"
+}
+
+param "image-tag" {
+  default = "latest"
+}
+
+docker.image "converge" {
+  name = "{{param `image-name`}}"
+  tag  = "{{param `image-tag`}}"
+}
+
+task.query "rpc-token" {
+  query = "uuid"
+}
+
+docker.container "converge" {
+  name  = "converge-test"
+  image = "{{lookup `docker.image.converge.name`}}:{{lookup `docker.image.converge.tag`}}"
+
+  expose = ["4774"]
+
+  volumes = [
+    "{{param `converge-bin-dir`}}:/converge",
+    "{{env `PWD`}}/samples:/samples",
+    "{{env `PWD`}}/examples:/examples",
+  ]
+
+  entrypoint = ["/converge/converge"]
+  command    = ["server", "--rpc-token", "{{lookup `task.query.rpc-token.status.stdout`}}"]
+}
+
+task.query "wait" {
+  query   = "sleep 10"
+  depends = ["docker.container.converge"]
+}
+
+task.query "container-port" {
+  query   = "docker inspect {{lookup `docker.container.converge.name`}} | jq '.NetworkSettings.Ports[\"4775/tcp\"][0].HostPort'"
```

Comment:
Is there a way to escape the text/template input for when converge loads the command? It was giving me trouble before.